ocd-954.sql

```diff
@@ -0,0 +1,11 @@
+﻿-- Add ONC Staff role to user_permission table to support API ROLE_ONC_STAFF
+INSERT INTO openchpl.user_permission(
+            user_permission_id, name, description, authority, creation_date, 
+            last_modified_date, last_modified_user, deleted)
+    SELECT 7, 'ONC STAFF', 
+    'This permission gives a user access to the CMS Download file and report navigation section. It denies editing of Users/Products/CPs/ACBs/etc. No user invitation ability.', 
```

Comment:
Does this say `ROLE ONC STAFF` or `ROLE_ONC_STAFF`? The underscores are important